arc/cli/commands/run.py

```
@@ -0,0 +1,478 @@
+
+
+def _estimate_cost(scenario_count: int, model: str) -> float:
+    """Estimate cost for running scenarios."""
+    # Simplified cost estimation
+    cost_per_scenario = {
+        "openai/gpt-4.1": 0.0010,
+        "openai/gpt-4.1-mini": 0.0002,
+        "anthropic/claude-opus-4": 0.0015,
+        "anthropic/claude-sonnet-4": 0.0005,
+    }
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)